cpp/src/arrow/compute/exec/expression_test.cc

```diff
@@ -476,15 +476,16 @@ TEST(Expression, BindLiteral) {
 }
 
 void ExpectBindsTo(Expression expr, util::optional<Expression> expected,
-                   Expression* bound_out = nullptr) {
+                   Expression* bound_out = nullptr,
+                   Schema* schema = kBoringSchema.get()) {
```

Comment:
I you could keep a const& here
```suggestion
                   const Schema& schema = *kBoringSchema) {
```

Reply:
Done.